<launch>

  <arg name = "segmentation_service" value="segmentation_service" />

  <node name ="segmentation_server" pkg="sq_fitting" type="segmentation_server_node" output="screen">
      <param name="zmax" value="2.0" />
      <param name="zmin" value="0.02" />
      <param name="th_points" value="400" />

      <!--Supervoxel parameters -->
      <param name="disable_transform" value = "true" />
      <param name="voxel_resolution" value="0.0075" />
      <param name="seed_resolution" value="0.03" />
      <param name="color_importance" value="0.0" />
      <param name="spatial_importance" value="1.0" />
      <param name="normal_importance" value="4.0" />

      <!-- LCCP parameters -->
      <param name="use_extended_convexity" value="false" />
      <param name="use_sanity_criterion" value="true" />
      <param name="concavity_tolerance_threshold" value="10" />
      <param name="smoothness_threshold" value="0.1" />
      <param name="min_segment_size" value="3" />

      <param name="segmentation_service" value="$(arg segmentation_service)"/>

  </node>



</launch>
